ocamlbuild.install

```diff
@@ -0,0 +1,4 @@
+bin: [
+  "?ocamlbuild.native" {"ocamlbuild"}
+  "ocamlbuild.byte" {"ocamlbuild.byte"}
+]
```

Comment:
If I understand correctly the [.install documentation](https://opam.ocaml.org/doc/Manual.html#packagenameinstall), the semantics of this description is that if ocamlbuild.native exists, then it is installed as "ocamlbuild", and ocamlbuild.byte is always installed as ocamlbuild.byte.

This is a bit disappointing: we would like users with no native compiler available to have an `ocamlbuild` executable available, named `ocamlbuild.byte`. I guess that we will need some sort of configuration layer producing the `ocamlbuild.install` file based on the detected native support.